### PR TITLE
Js Todo App

### PR DESCRIPTION
**Pull request by Davidosky007**
## Js Todo App
In this project 
* classes were used to dynamically generate objects
* properties like due date, priorities, description and title were added 
* delete functionality was added 
* Local storage was useed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+* Npm
+* file loader
+* style loader
+* css loader
+
+### Install and Setup
+
+* Clone repo to your local `https://github.com/Davidosky007/Js_todo_app.git`
+* cd into the cloned repo
+* Open a terminal in the root of the app
```
Comment (whiz25) [inline README.md]: - These instructions are not clear. What do I do next after changing the directory? How do I open the app?
- Always leave no room for assumptions. Give explicit commands that the user can use to start the app. In my case, I cloned and opened the app in `vscode` then ran `live-server dist` to open the `index.html` file located in the `dist` folder.
- Update this instruction

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Amazing work @Davidosky007. You did a great job. Your app works well and this PR checks 99% of the requirements needed.
- Codebase is modularised
- DOM logic is separated from business logic
- I was able to create, edit and delete both tasks and projects. Well done!
#### Changes
- Update the readme. Check my comments for suggestions


### Martin Afani
#### Slack: whiz25

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change Required✖️

# 👋 @Davidosky007 ,

* [ ] As mentioned in the previous review, let give clear instruction in our readme, here the link the [Microverse template](https://github.com/microverseinc/readme-template), Let add an image of the application. Let add explicitly what the user should do, they may not be necessarily tech savvy let be as clear as possible.

#### Let us address the issue and submit a re-review.

#### Happy coding👨‍💻

Addo Forison

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @Davidosky007  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implement the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] Your project will be professional when all of info presented on readme file are accurate, intead of being `Change to the project directory by entering cd "task-todo" in the terminal` it should be `Change to the project directory by entering cd "Js_todo_app" in the terminal`
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @Davidosky007  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**